src/preprocessor.ts

```diff
@@ -1,10 +1,18 @@
+// @ts-ignore
+import findBabelConfig from 'find-babel-config';
+import * as p from 'process';
+import merge from 'deepmerge';
 import { parse as parser, ParserOptions } from '@babel/parser';
-import { merge } from 'lodash';
-import { loadPartialConfig } from '@babel/core';
 import traverse, { NodePath } from '@babel/traverse';
 import { ImportDeclaration } from '@babel/types';
 import { PrettierParserOptions, getCodeFromAst, getSortedNodes } from './utils';
 
+export function getBabelConf() {
```

Comment:
We do not need to export it. Let's move this function to `get-babel-conf.ts` file.